lib/monero/wallet.ex

```diff
@@ -68,6 +68,43 @@ defmodule Monero.Wallet do
     request("open_wallet", %{filename: filename, password: password})
   end
 
+  @type transfer_destination :: %{amount: String.t, address: String.t}
+
+  @type transfer_opts :: {:payment_id, String.t}
+    | {:get_tx_key, boolean}
+    | {:priority, non_neg_integer}
+    | {:do_not_relay, boolean}
+    | {:get_tx_hex, boolean}
+
+  @doc """
+  Send monero to a number of recipients.
+
+  Args:
+  * `destinations` - List of destinations to receive XMR.
+  * `mixin` - Number of outpouts from the blockchain to mix with (0 means no mixing).
+  * `unlock_time` - Number of blocks before the monero can be spent (0 to not add a lock)
+  * `payment_id` - (Optional) Random 32-byte/64-character hex string to identify a transaction.
```

Comment:
maybe better to specify the list `transfer_opts` with all optional args? like
```
* `transfer_opts` - List of optional argument:
  * `payment_id` - (Optional) ...
```
etc?

Reply:
I agree - keeping docs in sync is not something that we want to spend our time on